lib/portage/tests/runTests.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
Drop again? Looks like a merge conflict error (we don't want runTests.py now, right?)

Reply:
-> https://github.com/gentoo/portage/pull/1146#discussion_r1369429609